client/state/products-list/actions.js

```diff
@@ -23,6 +27,140 @@ export function receiveProductsList( productsList ) {
 		] )
 	);
 
+	// TODO: remove this once products are added to the Store.
```

Comment:
Bringing @Automattic/jetpack-design to make double sure about product naming: are we using "Backup" and "Backup Pro"? I know at one point we were looking at an alternate form like "Backup (200 GB)" and "Security (2 TB)."

Reply:
Also: are we showing both "tiers" of products on the pricing page, or just the basic tier of each one with a "starting at $XX"-style price listing?

Reply:
> are we using "Backup" and "Backup Pro"? I know at one point we were looking at an alternate form like "Backup (200 GB)" and "Security (2 TB)."

I got the names from the Figma, which has different names on the main pricing page and the backup pricing page. So I was going to alter the names for the backup pricing page.